WordPress/src/main/java/org/wordpress/android/ui/comments/EditCommentActivity.java

```diff
@@ -95,14 +94,9 @@ private void loadComment(Intent intent) {
 
             configureViews();
         } else {
-            if (SimperiumUtils.getNotesBucket() != null) {
-                try {
-                    mNote = SimperiumUtils.getNotesBucket().get(noteId);
-                    requestFullCommentForNote(mNote);
-                } catch (BucketObjectMissingException e) {
-                    SimperiumUtils.trackBucketObjectMissingWarning(e.getMessage(), noteId);
-                    showErrorAndFinish();
-                }
+            mNote = NotificationsTable.getNoteById(noteId);
+            if (mNote != null) {
+                requestFullCommentForNote(mNote);
             }
```

Comment:
I think we'd need to show the user something happened, if we can't manage to obtain the Note from our database.

We should be callling `showErrorAndFinish();` in an `else` statement here